packages/block-editor/src/components/url-popover/index.js

```diff
@@ -16,7 +16,7 @@ function URLPopover( {
 	additionalControls,
 	children,
 	renderSettings,
-	position = 'bottom center',
+	placement = 'bottom',
```

Comment:
Isn't this a breaking change? Should we still pass a deprecated `position` and pass it as a `popover` prop in case someone uses this component outside of the repo, but still relies on the legacy prop?

Reply:
Good point, I guess that approach was indeed introducing a breaking change!

I've pushed some changes and adopted a more conservative approach — see the updated PR description for more details